### ISSUE TITLE
stop cheating

### ISSUE THREAD

**Issue by tpoisonooo** [ZHZisZZ/weak-to-strong-search#1]
If you are not ready to open source your code, please **do noooot** creat it .

**Comment by tpoisonooo** [comment]
you said your source code is here and leave a blank.

![image](https://github.com/ZHZisZZ/weak-to-strong-search/assets/7872421/5af7cfad-30ab-42a6-bead-88a04e792272)

**Comment by tpoisonooo** [comment]
![image](https://github.com/ZHZisZZ/weak-to-strong-search/assets/7872421/1819c55c-b5e7-4ff7-808e-c16e1dc071ea)

**Comment by ZHZisZZ** [comment]
I am just wrapping up the ACL camera-ready papers. Our code will be open-sourced later, for your enlightenment. By the way, could you kindly cease being rude and ignorant?

**Comment by ZHZisZZ** [comment]
Why do you believe using placeholder link constitutes cheating? A public GitHub repository is an excellent platform for gathering constructive feedback from the public, rather than receiving ungrounded claims from you (i.e., cheating). This feedback helps us improve our work before its official release. TBA clearly conveys that we will upload our code as soon as possible (after I wrap up some camera ready papers). Using a placeholder repository is a common practice; for instance, the [Alignment Handbook repository](https://github.com/huggingface/alignment-handbook) serves as a placeholder for several months until their codes are finalized. **If I have violated the standards of Shanghai AI Lab, I am willing to accept your criticism. Otherwise, you should be accountable for your statements as a self-proclaimed representative of Shanghai AI Lab.**

### By the way, stop editing your messages significantly as if it is me who started this argument.

**Comment by tpoisonooo** [comment]
> Why do you believe using placeholder link constitutes cheating? 

Yes, github placeholder is an excellent platform for gathering feedback about paper, I never think about it before. 
My main concern is whether the implementation exists or not.

“stop cheating” is indeed rude expression. 

> self-proclaimed representative

I have reviewed my word, it seems there has been no indication of claiming to be a representative of anything.

I simply stated that I am also a reader, and when I see code, I will check which organization it belongs to, which is a very normal behavior.

Everyone has the day they leave here, it's a job, no special meaning.

**Comment by tpoisonooo** [comment]
For gathering feedback, we usually create a group and immediately send messages after open-sourcing the code. It's faster than GitHub.

**Comment by tpoisonooo** [comment]
> Using a placeholder repository is a common practice

Yes it is a common practice, common practice != rightness. 
Each team has their own principle, so I said, `I feel it's quite inappropriate to judge other people's work.`

**Comment by tpoisonooo** [comment]
> Why do you believe using placeholder link constitutes cheating?

It seems  the core disagreement.

* I asked myself again again and again, yes using placeholder link is cheating, I would not do it before or later. **I hope I won't kill the me of now**
* sorry for my rude expression and judge your work

**Comment by tpoisonooo** [comment]
> as if it is me who started this argument.
No. I start this argument.

**Comment by ZHZisZZ** [comment]
Please don't try to tell me what to do if you're not in a position to set the rules for the whole lab. If you knew my affiliations, you should have conveyed your opinions in a private channel. But since you've brought it up publicly, so be it.

**Comment by tpoisonooo** [comment]
> Please don't try to tell me what to do 

Since I not in a position to set rules, so I closed the issue before, you reopen it.

**Comment by tpoisonooo** [comment]
Can we close it now ?
